src/mono/tests/HwIntrinsics/Program.cs

```diff
@@ -38,17 +38,17 @@ static void Main ()
         Console.WriteLine ();
 
         if (sse1Methods != 92)
```

Comment:
Is this test running on CI?

Reply:
no, it was ported from mono/mono and we didn't have time to hook it up yet